Assignment2/DanielMiM-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,91 @@
+{
+    "@context": {
+        "inf": "http://example.org",
+        "subj": "http://example.org/subject",
```

Comment:
Be careful whit prefix definitions, the URI must have a / or # at the end. -> "http://example.org/"